lib/src/tools/test_tool.dart

```diff
@@ -186,10 +186,10 @@ List<String> buildArgs({
   ArgResults argResults,
   List<String> configuredBuildArgs,
   List<String> configuredTestArgs,
-  bool useBuildTest,
```

Comment:
fyi only the `TestTool` class in this file is publicly exported, so this param name change is safe.